modules/Emails/include/ListView/ListViewDataEmails.php

```diff
@@ -269,209 +191,817 @@ public function getListViewData(
+                                $this->seed->assigned_user_id = $temp->assigned_user_id;
+                            }
+                            if($idIndex[$row[$id_field]][0] == $dataIndex){
+                                $pageData['tag'][$dataIndex] = $temp->listviewACLHelper();
+                            }else{
+                                $pageData['tag'][$dataIndex] = $pageData['tag'][$idIndex[$row[$id_field]][0]];
+                            }
+                            $data[$dataIndex] = $temp->get_list_view_data($filter_fields);
+                            $detailViewAccess = $temp->ACLAccess('DetailView');
+                            $editViewAccess = $temp->ACLAccess('EditView');
+                            $pageData['rowAccess'][$dataIndex] = array('view' => $detailViewAccess, 'edit' => $editViewAccess);
+                            $additionalDetailsAllow = $this->additionalDetails && $detailViewAccess && (file_exists(
+                                        'modules/' . $temp->module_dir . '/metadata/additionalDetails.php'
+                                    ) || file_exists('custom/modules/' . $temp->module_dir . '/metadata/additionalDetails.php'));
+                            $additionalDetailsEdit = $editViewAccess;
+                            if($additionalDetailsAllow) {
+                                if($this->additionalDetailsAjax) {
+                                    $ar = $this->getAdditionalDetailsAjax($data[$dataIndex]['ID']);
+                                }
+                                else {
+                                    $additionalDetailsFile = 'modules/' . $this->seed->module_dir . '/metadata/additionalDetails.php';
+                                    if(file_exists('custom/modules/' . $this->seed->module_dir . '/metadata/additionalDetails.php')){
+                                        $additionalDetailsFile = 'custom/modules/' . $this->seed->module_dir . '/metadata/additionalDetails.php';
+                                    }
+                                    require_once($additionalDetailsFile);
+                                    $ar = $this->getAdditionalDetails($data[$dataIndex],
+                                        (empty($this->additionalDetailsFunction) ? 'additionalDetails' : $this->additionalDetailsFunction) . $this->seed->object_name,
+                                        $additionalDetailsEdit);
+                                }
+                                $pageData['additionalDetails'][$dataIndex] = $ar['string'];
+                                $pageData['additionalDetails']['fieldToAddTo'] = $ar['fieldToAddTo'];
                             }
+                            next($rows);
+                        }
+                    }
+                    $nextOffset = -1;
+                    $prevOffset = -1;
+                    $endOffset = -1;
```

Comment:
Not always. when you first look at the list view with out setting up an email account you need these values to be set.